Fix #1127 Schema->hasTable and Schema->hasView does not support yii-quoted names (#1128)
yiisoft/db: CHANGELOG.md

## 2.0.1 under development

> - Bug #1127: Fix `AbstractSchema::hasTable` and `AbstractSchema::hasView` did not support yii-quoted `{{%table}}` and `{{%view}}` names (@batyrmastyr)

Reviewer: ```suggestion
- Bug #1127: Fix `AbstractSchema::hasTable()` and `AbstractSchema::hasView()` methods to support names quoted with curly brackets `{{%table}}` (@batyrmastyr)
```
